apps/judicial-system/web/src/routes/Shared/Cases/PrisonCases.tsx

```diff
@@ -211,6 +218,20 @@ export const PrisonCases: FC = () => {
             {
               cell: (row) => <ColumnCaseType type={row.type} />,
             },
+            {
+              cell: (row) => {
+                const punishmentType = isNonEmptyArray(row.defendants)
```

Comment:
Same as above, but as this is temporary code it does not really matter.

Reply:
Same as [above](https://github.com/island-is/island.is/pull/17285#discussion_r1892161450)